doc/api/assert.markdown

```diff
@@ -409,3 +408,4 @@ assert.throws(
 [`Error`]: errors.html#errors_class_error
 [`RegExp`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
 [`TypeError`]: errors.html#errors_class_typeerror
+[Locked]: documentation.html#documentation_stability_index
```

Comment:
Something just doesn't seem right, [Lexicographic](https://en.wikipedia.org/wiki/Lexicographical_order) on Wikipedia for example.

> the lexicographic or lexicographical order (also known as lexical order, dictionary order, alphabetical order or lexicographic(al) product) is a generalization of the way the alphabetical order of words is based on the alphabetical order of their component letters.

This isn't ordered by the ASCII values as `A` (#65) comes before `a` (#97) in ASCII values. As an example:

```js
['Locked', 'assert.deepEqual()', 'assert.deepStrictEqual()', 'assert.ok()','Error','RegExp', 'TypeError'].sort();
// ["Error", "Locked", "RegExp", "TypeError", "assert.deepEqual()", "assert.deepStrictEqual()", "assert.ok()"]

'RegExp' > 'Locked';
// true
```

I'm not trying to hold this PR up, just lost as to how this ordering occurred and wondering if the other PRs are in an odd order as well.